test/__snapshots__/mathml-spec.js.snap

```diff
@@ -204,6 +204,211 @@ exports[`A MathML builder normal spaces render normally 1`] = `
 </math>
 `;
 
+exports[`A MathML builder should build the CD environment properly 1`] = `
+<math xmlns="http://www.w3.org/1998/Math/MathML"
+      display="block"
+>
+  <semantics>
+    <mtable rowspacing="0.24999999999999992em"
```

Comment:
nit: why not `0.25em`?

Reply:
It's a computed number from https://github.com/KaTeX/KaTeX/blob/master/src/environments/array.js#L514-L519.

I'll add code that will round it to 4 places after the decimal.

Reply:
I think it's fine to leave this as is for now and handle it as part of #2460.